### PR TITLE
[12.0][ADD] delivery: Migration scripts

### PR DESCRIPTION
**Pull request by pedrobaeza**
cc @Tecnativa

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
You should add the `noupdate_changes.xml` 😉

**Comment by pedrobaeza** [comment]
Indeed! Done

File: addons/delivery/migrations/12.0.1.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+from psycopg2.extensions import AsIs
```
Comment (MiquelRForgeFlow) [inline addons/delivery/migrations/12.0.1.0/post-migration.py]: flake8 wants you to delete this unused line

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request closed without merging**